source/overview/license-and-subscription.rst

```diff
@@ -0,0 +1,126 @@
+Licensing and Subscription
+----------------------
+
+Mattermost offers a secure self-service portal where you can manage your Mattermost Enterprise subscriptions for your on-premises deployments.
+All your licenses are listed in the portal, including their start and end dates, providing you with an at-a-glance overview of your account, making it easier to manage purchases and renewal dates.
+
+You can access your account to view information about your:
```

Comment:
Wherever we say "account" in this page, we should change it to "portal account" to make it clear we're not referring to a product account.

Reply:
Excellent point, will get this done today.